### PR TITLE
[HOPSWORKS.AI-DOC] Fix EKS screenshots

### PR DESCRIPTION
**Pull request by maismail**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -20,12 +20,12 @@ nav:
       - Azure: hopsworksai/azure/restrictive_permissions.md
     - Cluster creation:
       - AWS: hopsworksai/aws/cluster_creation.md
+      - AWS (EKS and ECR): hopsworksai/aws/eks_ecr_integration.md
```
Comment (berthoug) [inline mkdocs.yml:23]: Can you rename to `Integration with EKS and ECR` right now I find it confusing to have two AWS

File: docs/hopsworksai/aws/eks_ecr_integration.md [hunk 2]
```diff
@@ -316,40 +316,39 @@ Choose your preferred SSH key to use with the cluster, then click Next:
   </figure>
 </p>
 
-Choose *Enabled* to enable the use Amazon EKS and ECR:
+Choose the instance profile role that you have created in Step 2, then click Next:
 
 <p align="center">
   <figure>
     <a  href="../../../assets/images/hopsworksai/aws/eks-hopsworks-create-cluster-1.png">
       <img src="../../../assets/images/hopsworksai/aws/eks-hopsworks-create-cluster-1.png" alt="Choose Enabled">
```
Comment (berthoug) [inline docs/hopsworksai/aws/eks_ecr_integration.md:324]: This is not the screenshot for the instance profile

**Review by berthoug — CHANGES_REQUESTED** [review]

File: mkdocs.yml (hunk 1 above)
Comment (maismail) [reply mkdocs.yml]: Fixes

File: docs/hopsworksai/aws/eks_ecr_integration.md (hunk 2 above)
Comment (maismail) [reply docs/hopsworksai/aws/eks_ecr_integration.md]: Fixed

**Review by berthoug — APPROVED** [review]

**Review by moritzmeister — APPROVED** [review]

**Pull request merged**